QModManager/Utility/IOUtilities.cs

```diff
@@ -31,7 +31,7 @@ internal static string GetFolderStructureAsTree(string directory = null)
             }
         }
 
-        internal static string GenerateFolderStructure(string directory)
+        private static string GenerateFolderStructure(string directory)
```

Comment:
(note) A lot of `internal` things can be made `private` as long as they're not used in Unit Tests